framework/wazuh/core/indexer/commands.py

```diff
@@ -1,13 +1,13 @@
 from dataclasses import asdict
-from typing import List, Optional
+from typing import List, Optional, Union
 
-from opensearchpy import NotFoundError
 from uuid6 import UUID
 
 from .base import BaseIndex, IndexerKey, remove_empty_values
-from wazuh.core.indexer.models.commands import Command, Status
 from wazuh.core.exception import WazuhResourceNotFound
+from wazuh.core.indexer.models.commands import Command, Result, Status
 
+ID_KEY = 'id'
```

Comment:
This could be within the `IndexerKey`

Reply:
Same as in https://github.com/wazuh/wazuh/pull/25262#discussion_r1725471934